docs/network protocol.md

```diff
@@ -191,11 +193,24 @@ Sent to clients after a client requested this message be sent to them, more info
 ### InvalidPacket
 Sent to clients if the server caught a problem with a packet. This only occurs for errors that are explicitly checked for.
 
+### Retrieved
+Sent to clients as a response the a [Get](#Get) package
+#### Arguments
+| Name | Type | Notes |
+| ---- | ---- | ----- |
+| keys | Dict\[str\, any] | a Key-value collection containing all the values for the keys requested in the [Get](#Get) package |
```

Comment:
```suggestion
| keys | dict\[str\, any] | A key-value collection containing all the values for the keys requested in the [Get](#Get) package. |
```

Reply:
I think the `]` after `any` should also be escaped, so `dict\[str\, any\]`